pygmt/helpers/decorators.py

```diff
@@ -322,6 +323,30 @@ def fmt_docstring(module_func):
     return module_func
 
 
+def insert_alias(module_func, default_value=None):
+    """
+    Decorator insertings aliases into the signature of a method.
```

Comment:
I'm thinking that this function might be best left for internal use only, so maybe put an underscore in the front (see https://stackoverflow.com/a/8689983 for naming convention)? 

```suggestion
def _insert_alias(module_func, default_value=None):
    """
    Function to insert PyGMT long aliases into the signature of a method.
```